modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -166,6 +170,25 @@ public void deleteUserAccountsByEmailAddress(
 		}
 	}
 
+	@Override
+	public Organization getAccountByExternalReferenceCodeOrganization(
+			String externalReferenceCode, String organizationId)
+		throws Exception {
+
+		AccountEntry accountEntry =
+			_accountEntryService.fetchAccountEntryByExternalReferenceCode(
+				contextCompany.getCompanyId(), externalReferenceCode);
+
+		AccountEntryOrganizationRel accountEntryOrganizationRel =
+			_accountEntryOrganizationRelService.
+				fetchAccountEntryOrganizationRel(
+					accountEntry.getAccountEntryId(),
+					Long.valueOf(organizationId));
+
+		return _toOrganization(
+			String.valueOf(accountEntryOrganizationRel.getOrganizationId()));
```

Comment:
This will be an exception if the fetchAccountEntryOrganizationRel() returns null.
Either we manage the null value or we use a get instead of a fetch